messages.go

```diff
@@ -7,16 +7,33 @@ import (
 	"net/http"
 )
 
+type ProvidersChannelResponse struct {
+	Key string
+	Template string
```

Comment:
There is a Name of type string. This would appear if you went to the Channel settings and in the General Tab, change the value under __Rename__.

Reply:
Added thanks!